qemu/tests/boot_nbdimage_with_qsd.py

```diff
@@ -0,0 +1,69 @@
+import os
+
+from virttest import data_dir
+from virttest import qemu_storage
+
+from avocado import fail_on
```

Comment:
Unused fail_on imported from avocado

Reply:
Suggest you run the following commands, before pushing the patch.
`# pycodestyle qemu/tests/boot_nbdimage_with_qsd.py 
`# pyflakes qemu/tests/boot_nbdimage_with_qsd.py

Reply:
Thanks for the info.